model_analyzer/record/record.py

```diff
@@ -24,14 +24,34 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import time
+from abc import ABC, abstractmethod
 
 
-class Record:
+class Record(ABC):
     """
     This class is used for representing monitoring
     records
     """
 
-    def __init__(self, name):
-        self.time = time.time()
```

Comment:
What about the record times?

Reply:
Added a timestamp abstract method that must be implemented.